site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -951,6 +951,29 @@ components:
                   notice.
             "ext":
                $ref: '#/components/schemas/Extensions'
+            "status":
+               $ref: '#/components/schemas/Status'
+
+      Status:
+         type: object
+         additionalProperties: false
+         description: Optional and written only by the device registry implementation.
+            It can not be updated by the user.
+            Any data inserted in the status section by the user in a PUT request will be ignored.
+         properties:
+            "created":
+               type: string
+               format: date-time
+               description: The date and time when the device was created.
+                  The format must comply with RFC 3339, section 5.6. The representation must be UTC (no timezone offset).
+            "updated":
+               type: string
+               format: date-time
+               description: The date and time when the device was last updated.
+                  The format must comply with RFC 3339, section 5.6. The representation must be UTC (no timezone offset).
+            "last-user":
+               type: string
+               description: The user who made the last edit on this device.
```

Comment:
please add the status to the example payloads for the device management endpoint as well ...